package.json

```diff
@@ -1,8 +1,8 @@
 {
     "private": true,
     "scripts": {
-        "dev": "npm run development",
-        "development": "mix",
+        "dev": "npm run development -- --watch",
+        "development": "node_modules/cross-env/src/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --config=node_modules/laravel-mix/setup/webpack.config.js",
```

Comment:
I believe doing the development script like this is side stepping mix? I think we need to change the package manager in the mix file to use Yarn instead,  see https://github.com/JeffreyWay/laravel-mix/pull/2642 I think they added yarn support officially in Mix 6.X

Reply:
done